app/templates/play/campaign-view.jade

```diff
@@ -148,7 +148,10 @@ if view.showAds()
                   button.btn.btn-primary.btn.btn-lg.btn-block.btn-illustrated
                     span(data-i18n="common.play") Play
               else
-                button.btn.btn-success.btn.btn-lg.btn-illustrated.start-level(data-i18n="common.play") Play
+                if level.requiresSubscription && !me.isPremium()
```

Comment:
We should use the same logic as `onClickStartLevel` uses to `@promptForSubscription` in `CampaignView.coffee`.  Probably easiest to pull that logic into a separate function and call from both places.